apps/src/redux/instructions.js

```diff
@@ -321,6 +329,7 @@ export const determineInstructionsConstants = config => {
     shortInstructions2,
     longInstructions,
     teacherMarkdown,
-    hasContainedLevels
+    hasContainedLevels,
+    levelVideo
```

Comment:
Right now, this will be true or undefined. Is that the intention, or should it be true/false? Either way, we can do our assignment here, rather than assigning to a variable that we don't use elsewhere, i.e.
```
// true/false (i suspect this is what we expect)
levelVideo: !!config.level.videoKey,

// true/undefined
levelVideo: !!config.level.videoKey || undefined
```